### PR TITLE
Feature/82 turret tracking

### PR DESCRIPTION
**Pull request by JTMongeon**
Merge

### CODE REVIEW

**Review by JTMongeon — COMMENTED** [review]
test

File: src/main/java/frc/shooter/commands/LimelightTurret.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package frc.shooter.commands;
+
+import frc.drives.DrivesSensorInterface;
+import frc.shooter.ShooterCommand;
+import frc.shooter.ShooterOutput;
+import frc.shooter.ShooterSensorsInterfeace;
+
+public class LimelightTurret extends ShooterCommand {    
+	double deadband = 2;
+    double tx;
```
Comment (secretspy97) [inline src/main/java/frc/shooter/commands/LimelightTurret.java]: No need for this to be a class variable

File: src/main/java/frc/shooter/commands/LimelightTurret.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    public LimelightTurret(ShooterSensorsInterfeace sensors, DrivesSensorInterface driveSensors){
+        super(sensors, driveSensors);
+    }
+
+    @Override
+    public ShooterOutput execute(){
+        tx = sensors.getAngleToTarget();
+        double speed = 0; 
+        if(Math.abs(tx) > deadband) {
```
Comment (secretspy97) [inline src/main/java/frc/shooter/commands/LimelightTurret.java]: With a P-Loop you shouldn't need a deadband. Will have to test

**Review by secretspy97 — CHANGES_REQUESTED** [review]
Need to add Tests. See test.frc.drives.commands for test examples

**Review by eddy46 — APPROVED** [review]

**Pull request merged**